### ISSUE TITLE
Design discussion: How to run training from hi-ml

### ISSUE THREAD

**Issue by ant0nsc** [microsoft/hi-ml#104]
high-level goal: Port the bring-your-own-model framework over from InnerEye

Check with Stephanie and Javier about what they'd need to have their project support

Functionality:
* Select model by name
* Setup the container
* Create the trainer
* Run training
* Run evaluation on test set
* Run reporting

We need find a good way of telling the runner where the models live - models should live in the user's repository, but the hi-ml training runner needs to know where that is
One possible solution: users need to define their own training script, define a folder where their models live, and then pass on to the hi-ml trainer
